src/services/index.ts

```diff
@@ -0,0 +1,2 @@
+export { AnalyticsService } from './AnalyticsService';
+export { HttpRequesterService } from './HttpRequesterService';
```

Comment:
Does HttpRequesterService need to be exported? I didn't think it would be accessed by users of the library

Reply:
The HttpRequesterService would need to be exported if we make `AnalyticsReporter` public, however since we are now making it internal, `HttpRequesterService` can be internal as well